engine/src/main/java/org/terasology/rendering/dag/tasks/DisableCapabilityTask.java

```diff
@@ -32,4 +33,9 @@ public DisableCapabilityTask(int capability) {
     public void execute() {
         glDisable(this.capability);
     }
+
+    @Override
+    public String toString() {
+        return String.format("%21s: disabled", SetCapability.getCapabilityName(capability));
```

Comment:
I just focused for the first time on what is being done in this and other similar method. I'm concerned about this way of doing things because it's hardly extensible. But let's leave it as it is for now. We'll get back to it at a different stage.